dali/test/python/test_segmentation_utils.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
```

Comment:
Nitpick:
The filename sounds like it's testing segmentation utils. If these are utilities for testing segmentation, then the correct name would be `segmentation_test_utils.py`.

Reply:
done